_posts/en/newsletters/2024-01-24-newsletter.md

```diff
@@ -0,0 +1,189 @@
+---
+title: 'Bitcoin Optech Newsletter #286'
+permalink: /en/newsletters/2024/01/24/
+name: 2024-01-24-newsletter
+slug: 2024-01-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Disclosure of fixed consensus failure in btcd:** "Niklas Gögge
+  used Delving Bitcoin to [announce][gogge btcd] a consensus failure in
+  older versions of btcd that he had previously [responsibly
+  disclosed][topic responsible disclosures].  Relative time locks were
+  added to Bitcoin in a soft fork by adding consensus-enforced meaning
+  to transaction input sequence numbers.  To ensure any presigned
+  transactions created before the fork would not be made invalid, the
+  relative time locks only apply to transactions with version numbers 2
+  and higher, allowing transactions with the orginal default of version
+  1 to remain valid with any inputs.  However, in the original Bitcoin
+  protocol, version numbers are signed integers, meaning negative
```

Comment:
I think it's incorrect to say that version numbers are signed integers in the Bitcoin protocol. They are first and foremost a fixed-length byte sequence, and the only time they impact consensus they're treated as unsigned integers, so it's more a matter of how they're handled in the code.
```suggestion
  1 to remain valid with any inputs.  However, both Bitcoin Core and
  btcd treat version numbers as signed integers, meaning negative
```